daml/Marketplace/Trading/Service.daml

```diff
@@ -285,14 +250,14 @@ template CreateOrderRequest
         do
           exerciseByKey @Service serviceKey AcknowledgeOrderRequest with createOrderRequestCid = self, ..
 
-      nonconsuming RejectRequest : (ContractId Order.T, Order.TradeCollateral)
+      nonconsuming RejectRequest : ContractId Order.T
         with
```

Comment:
Again, question unrelated to this PR: What's the reason we need this and the below choices on this template (as a mere proxy to the choices on the service)?